src/subscriptions/ManageSubscriptionsPage.test.jsx

```diff
@@ -31,33 +29,20 @@ describe('<ManageSubscriptions />', () => {
       };
 
       jest.useFakeTimers();
-      const tree = renderer
-        .create(
-          <IntlProvider locale="en">
-            <Provider store={mockStore(storeMocksWithURL)}>
-              <ManageSubscriptionsPage />
-            </Provider>
-          </IntlProvider>,
-        )
-        .toJSON();
+
+      render(<ManageSubscriptionsPage />, storeMocksWithURL);
+
       act(() => {
         jest.runAllTimers();
```

Comment:
I think we can dispatch our actual saga routines here?
```
 act(() => {
      store.dispatch(
        fetchStripeCustomerPortalURL(
          subscriptionDetails, // I think we can either use mocks or factories here 
        ),
      );
      store.dispatch(fetchStripeCustomerPortalURL.fulfill());
    });
```    
   [factories](https://github.com/rosiejs/rosie)

Reply:
We are not using rosie in this mfe (for factories) and couldn't manage to accomplish this via mockstores